src/test/java/com/fasterxml/jackson/core/testutil/failure/JacksonTestFailureExpectedInterceptor.java

```diff
@@ -29,6 +29,39 @@ public void interceptTestMethod(Invocation<Void> invocation,
         handleUnexpectePassingTest(invocationContext);
     }
 
+    /**
+     * Interceptor API method that is called to intercept test template
+     * method invocation, like {@link org.junit.jupiter.params.ParameterizedTest}.
+     *<p>
+     * Before failing passing test case, it checks if the test should pass according to the
+     * {@link ExpectedPassingTestCasePredicate} provided in the
+     * {@link JacksonTestFailureExpected} annotation.
+     */
+    @Override
+    public void interceptTestTemplateMethod(Invocation<Void> invocation,
+            ReflectiveInvocationContext<Method> invocationContext, ExtensionContext extensionContext)
+        throws Throwable
+    {
+
+        try {
+            invocation.proceed();
+        } catch (Throwable t) {
+            // do-nothing, we do expect an exception
```

Comment:
Is there an expectation of always getting an exception, to indicate failing test? I guess this is true in that assert-methods do throw assertion exceptions.